src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLClip.m

```diff
@@ -58,7 +58,6 @@ @implementation MTLClip {
     BMTLSDOps*  _dstOps;
     BOOL _stencilMaskGenerationInProgress;
     BOOL _stencilMaskGenerationStarted;
-    BOOL _clipReady;
```

Comment:
This looks like cleanup and not related to this change. Sometime back i also had question on why this was needed. Thanks.